connectors/src/resources/slack_configuration_resource.ts

```diff
@@ -117,6 +117,19 @@ export class SlackConfigurationResource extends BaseResource<SlackConfigurationM
     );
   }
 
+  static async fetchByTeamId(slackTeamId: string) {
```

Comment:
Hm we might have a problem here, the resource has a `listForTeamId` method because SlacConfigurationModel is shared by both "slack" and "slack_bot". We want to make sure to retrieve only "slack" connector here

Reply:
Switched to listForTeamId as we discussed.